database/database/schema/mined_project_schema.py

```diff
@@ -3,6 +3,7 @@
 MinedProjectSchema = {
     'repo': NonEmptyStr,
     'latest_mined_version': NonEmptyStr,
+    'last_date_mined': NonEmptyStr,
```

Comment:
What format did you decide on?

Reply:
I've decided to use a string to represent the time that will be converted into a datetime object for better date comparison. The string format is, as an example, `Mon, 01 Jan 1970 00:00:00 GMT`. I found the API to be difficult with inserting a date value to be represented in MongoDB (uses ISODate wrapper).

Reply:
So, in the `artifacts` collection (https://github.com/BugSwarm/bugswarm/blob/a339ec9dae28f63fcf82112d59e888e7a4706364/database/database/schema/artifact_schema.py#L89) we store the time the artifact was inserted to the DB as a `RequiredInt` (representing the time since the Unix epoch) so we can filter via this field in the API.

Putting it as a string we would lose that functionality. Do we need this functionality?

Reply:
I don't believe we need this functionality. MongoDB's internal field `_updated` should be closely related to our new string field `last_mined_date` and you could use this field in a filter. Use cases I could think of for filtering via this field isn't significant in my opinion. Let me know if you want to discuss further.

Reply:
@octaviocorona I am not sure I understand. Are you saying we could use `_updated` or `last_mined_date` in the filter?

For either we cannot filter for dates when they are strings in our API.

Reply:
Sorry, you can only filter `_updated` through MongoDB, you are correct that we cannot filter for either through our API. I still think we do not need the functionality.

Reply:
So what is the pro/con of having it as a string vs having it as an int?

Reply:
I originally thought in terms of dates which is the result of the string as the `ISODate` cannot be inserted through our API.

The only con over a string vs an int is that we cannot filter by it. Ultimately I now believe I can change the new field to be an `int` and covert Unix epoch over to a datetime object to be used in the pipeline.

Reply:
Let's go that route then!

Reply:
@dtomassi Here are the changes 3e8d334ad923a8f28c02ce6c338dd60bec37beb2